vllm/v1/worker/gpu_input_batch.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         self.token_ids_cpu = self.token_ids_cpu_tensor.numpy()
         self.num_tokens = np.zeros(max_num_reqs, dtype=np.int32)
         self.num_prompt_tokens = np.zeros(max_num_reqs, dtype=np.int32)
-        self.num_computed_tokens_cpu = np.empty(max_num_reqs, dtype=np.int32)
```

Comment:
nit: file name can be changed to `input_batch.py` as it now already supports both tpu and gpu